src/eme.js

```diff
@@ -58,6 +59,15 @@ export const makeNewRequest = ({
 
     // based on https://www.w3.org/TR/encrypted-media/#example-using-all-events
     keySession.keyStatuses.forEach((status, keyId) => {
+      // Trigger an event so that other things can take action if appropriate.
+      // For instance, the `output-restricted` status should result in an
+      // error being thrown.
+      player.tech_.trigger({
```

Comment:
If we only need the tech, maybe it makes sense to pass down the tech, or just a generic event bus type object (as the code here doesn't need to know the specifics of where the event is being triggered).

Reply:
Yeah, that's a great suggestion. It looks like that's all `player` is used for everywhere in eme.js, so I guess I only need to pass the tech from [all the way back here](https://github.com/videojs/videojs-contrib-eme/blob/master/src/plugin.js#L185).